mRemoteV1/Config/Serializers/XmlConnectionsDeserializer.cs

```diff
@@ -487,6 +487,8 @@ private ConnectionInfo GetConnectionInfoFromXml(XmlNode xmlnode)
                 {
                     connectionInfo.SoundQuality = (ProtocolRDP.RDPSoundQuality)Tools.MiscTools.StringToEnum(typeof(ProtocolRDP.RDPSoundQuality), Convert.ToString(xmlnode.Attributes["SoundQuality"].Value));
                     connectionInfo.Inheritance.SoundQuality = bool.Parse(xmlnode.Attributes["InheritSoundQuality"].Value);
+                    connectionInfo.RDPMinutesToIdleTimeout = Convert.ToInt32(xmlnode.Attributes["RDPMinutesToIdleTimeout"].Value);
+                    connectionInfo.Inheritance.RDPMinutesToIdleTimeout = bool.Parse(xmlnode.Attributes["InheritRDPMinutesToIdleTimeout"].Value);
```

Comment:
Looking better - still need the null conditional operators. Trying to access the Value property on a null value will still throw an exception. The bonus is that this will fix the failing unit tests.

```csharp
collectionOfInts["abc"]?.Value
```